src/bcf/record.rs

```diff
@@ -354,7 +354,20 @@ impl Record {
         self.inner().n_allele()
     }
 
-    // TODO fn push_genotypes(&mut self, Genotypes) {}?
+    /// Add/replace genotypes in FORMAT GT tag.
+    ///
+    /// # Arguments
+    ///
+    /// - `genotypes` - a flattened, two-dimensional array of GenotypeAllele,
+    ///                 the first dimension contains one array for each sample.
+    ///
+    /// # Errors
+    ///
+    /// Returns error if GT tag is not present in header.
+    pub fn push_genotypes(&mut self, genotypes: &[GenotypeAllele]) -> Result<()> {
+        let encoded: Vec<i32> = genotypes.iter().map(|gt| i32::from(*gt)).collect();
+        self.push_format_integer("GT".as_bytes(), &encoded)
```

Comment:
Not really important, but we might as well use the bytes literal here:
```suggestion
        self.push_format_integer(b"GT", &encoded)
```